src/reducers/url-state.js

```diff
@@ -58,7 +64,10 @@ function selectedTab(state: string = 'calltree', action: Action) {
   }
 }
 
-function rangeFilters(state: StartEndRange[] = [], action: Action) {
+function rangeFilters(
+  state: StartEndRange[] = EMPTY_START_END_RANGES,
```

Comment:
not useful because it's the initial state. Then we return the same object `state` all the time.